cvmfs/monitor.h

```diff
@@ -35,12 +62,15 @@ class Watchdog : SingleCopy {
   /**
    * Crash cleanup handler signature.
    */
-  typedef void (*FnOnCrash)(void);
+  typedef void (*FnOnExit)(const bool crashed);
```

Comment:
Let's use an enum as an argument instead of a bool, so that we know what it means when calling it. Alternatively, always call with comment, like `on_exit_(true /* crashed */)`.